wandb/wandb_controller.py

```diff
@@ -353,19 +354,19 @@ def _log(self) -> None:
         self._logged += 1
 
     def _error(self, s: str) -> None:
-        print("ERROR:", s)
+        wandb.termlog(f"ERROR: {s}")
         self._log()
 
     def _warn(self, s: str) -> None:
-        print("WARN:", s)
+        wandb.termlog(f"WARN: {s}")
```

Comment:
same comment here for `termwarn`?

Reply:
i kept it log, because they already have the `WARN` prefix and i didn't to further deal with what it means

Reply:
termwarn adds a warning prefix, so it would be warnning: warn